UPGRADE_NOTES.md

```diff
@@ -1,3 +1,132 @@
+## Upgrading from 0.17 to 0.18
+
+- Ensure you've upgraded to crystal ????
```

Comment:
This depends on if we release this before the next version of crystal. This current release doesn't require the next crystal (yet).